security-response.md

```diff
@@ -70,76 +70,248 @@ vulnerability so they may start the patch, release and communication process.
+
+2. When all necessary changes are ready, scroll to the bottom of the advisory
+   page and click "Merge pull request(s)" to merge all open pull requests in the
+   temporary private fork
+
+Important notes about temporary private forks:
+
+- CI/CD integrations do not run in temporary private forks to maintain security
+- You cannot merge individual pull requests; all open PRs are merged together
+- Status checks will not run on PRs in temporary private forks
+- After merging changes, you can publish the advisory to alert the community
+
+For more details, see the [GitHub documentation on private forks](https://docs.github.com/en/code-security/security-advisories/working-with-repository-security-advisories/collaborating-in-a-temporary-private-fork-to-resolve-a-repository-security-vulnerability).
+
+### Getting Assistance
+
+If Maintainers need assistance with how to handle a security vulnerability:
+
+1. For general guidance of a non sensitive nature, Contact the Security SIG
+  through the #otel-sig-security channel in the Cloud Native Slack workspace.
+  Do not expose sensitive information in this channel as it isn't a secure
+  communication mechanism.
+
+2. For sensitive guidance specific to an Advisory, reach out to Security SIG
+  Maintainers as they have access to all Advisories and can comment directly in
+  them. For complex or high impact issues, the Security SIG may also involve
+  the TC and/or GC.
+
+### CVE Management
+
+#### When to Request a CVE
+
+When determining whether to request a CVE identifier, Maintainers should
+consider the type of OTel deliverables affected (executable vs. library) and the
+source of the vulnerability (direct vs. dependency):
+
+##### For Executables (Collector, Tools, etc.)
+
+1. **Direct Vulnerabilities**: Always request a CVE when a vulnerability exists
+   in OTel's own code that could expose users to security risks.
+
+2. **Dependency Vulnerabilities**: Request a CVE if:
+   - The executable uses the vulnerable part of the dependency
+   - The vulnerability can be exploited through the executable's usage patterns
+   - The vulnerability creates a security impact for users of the executable
+
+   Do not request a CVE if the vulnerability in the dependency cannot be
+   triggered through normal usage of the executable.
+
+   Note that the vulnerability impact for the OTel executable may be different
+   than for the dependency that caused it, depending on how it is used.  Do not
+   assume that the OTel CVE must have the same score as the dependency's CVE.
+
+##### For Libraries
+
+1. **Direct Vulnerabilities**: Request a CVE if the vulnerability exists in
+   OTel's own library code.
+
+2. **Dependency References**: Generally, do not request a CVE when:
+   - The library only references a vulnerable dependency
+   - The dependency is not bundled with the OTel deliverable
+   - The vulnerability only affects consumers who explicitly import both the
+     OTel library and the vulnerable dependency
```

Comment:
I suspect that most dependencies are brought in transitively without explicit import

Reply:
also, similar bullets as above probably apply to libraries:

- The executable uses the vulnerable part of the dependency
- The vulnerability can be exploited through the executable's usage patterns
- The vulnerability creates a security impact for users of the executable

Reply:
can we have a threshold (Medium or lower) for transitive dependencies that we don't even need to do the analysis for? (as long as we are rolling update into next monthly release)